pom.xml

```diff
@@ -120,13 +120,13 @@
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
-      <version>3.0.10</version>
-      <scope>provided</scope>
+      <version>3.3.0</version>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.maven.plugin-testing</groupId>
       <artifactId>maven-plugin-testing-harness</artifactId>
-      <version>3.1.0</version>
+      <version>3.3.0</version>
```

Comment:
so we updated prerequisites to Maven 3.2.5?

Reply:
Good question!